pkg/universe.dagger.io/alpha/uffizzi/image.cue

```diff
@@ -0,0 +1,15 @@
+package uffizzi
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	// The python version to use
+	version: *"2022-04-12" | string
```

Comment:
Let's use a newer version tag.

Reply:
Just updated this to `latest`

Reply:
`latest` is a moving target, which means that stuff might break and we will not know about it until it happens. We should pin to a specific version, just newer than `2022-04-12`. I'm seeing [0.11.2](https://hub.docker.com/layers/cli/uffizzi/cli/v0.11.2/images/sha256-81d0208ca500cc067c0991a4b4643a027f71d855f0a3d1a23a082383c5e461c1?context=explore), released 8 days ago → https://github.com/UffizziCloud/uffizzi_cli/releases . 

This is a microscopic change, so pushing it now so that we can merge right after the checks go green.